src/categorical_algebra/CSets.jl

```diff
@@ -643,18 +647,18 @@ function naturality_failures(X, Y, comps, type_comps; only_combinatorial=false)
   Fun = Union{SetFunction,VarFunction,LooseVarFunction}
   comps = Dict(a => isa(comps[a],Fun) ? comps[a] : FinDomFunction(comps[a])  
                for a in keys(comps))
-              
   type_comps = Dict(a => isa(type_comps[a], Fun) ? type_comps[a] : 
                         SetFunction(type_comps[a],TypeSet(X,a),TypeSet(Y,a)) 
                     for a in keys(type_comps))
-  α = merge(comps, only_combinatorial ? Dict() : type_comps)
```

Comment:
Is `only_combinatorial` still supported as a keyword argument? I don't see it being used in the new code.

Reply:
The integration tests passed (other than AlgRewriting, which has a compatible PR waiting to be released once this one is merged).

`only_combinatorial` was relevant when we considered the subtle difference of `ACSetTransformations` and `CSetTransformations` which, this past year, came to have the meaning of two different ways of composing the same data (the latter *ignored* attribute data). However, this was a distinction we tried to make at the type level because GATlab didn't exist! Now we can simply have two different models of ThCategory for ACSetTransformations, one which ignores attributes and one which doesn't. So I think we can go ahead and remove this (currently unused) functionality.